fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/api/RunreportsApiResource.java

```diff
@@ -105,6 +109,10 @@ public Response runReport(@PathParam("reportName") @Parameter(description = "rep
         // Pass through isSelfServiceUserReport so that ReportingProcessService implementations can use it
         queryParams.putSingle(IS_SELF_SERVICE_USER_REPORT_PARAMETER, Boolean.toString(isSelfServiceUserReport));
 
+        if (parameterType) {
+            return datatableReportingProcessService.processRequest(reportName, queryParams);
+        }
+
```

Comment:
Can you please point out what are we trying to do here?

Reply:
Seems to be related to issue https://issues.apache.org/jira/browse/FINERACT-1336. Is this issue related to SQL injection? I am good to merge this here..however, if they are not related it would be better to have a separate PR for this one? 

cc: @ptuomola @vorburger

Reply:
Seems to be related to issue https://issues.apache.org/jira/browse/FINERACT-1336

Reply:
correct, related to FINERATC-1336